test/fixtures/app/thrift/shared.thrift

```diff
@@ -29,6 +29,7 @@ namespace java shared
 namespace perl shared
 namespace php shared
 namespace haxe shared
+namespace ex shared
```

Comment:
I think I'd prefer `elixir` spelled out. `ex` is too ambiguous, especially when that's not the name of the language.